Sources/BraintreeCard/BTCardClient.swift

```diff
@@ -0,0 +1,218 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Used to process cards
+@objc public class BTCardClient: NSObject {
+
+    // MARK: - Internal Properties
+
+    /// Exposed for testing to get the instance of BTAPIClient
+    let apiClient: BTAPIClient
+
+    let graphQLTokenizeFeature: String = "tokenize_credit_cards"
+
+    // MARK: - Initializer
+
+    /// Creates a card client
+    /// - Parameter apiClient: An API client
+    @objc(initWithAPIClient:)
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+
+    // MARK: - Public Methods
+
+    /// Tokenizes a card
+    /// - Parameters:
+    ///    - card: The card to tokenize.
+    ///    - completion: A completion block that is invoked when card tokenization has completed. If tokenization succeeds,
+    ///    `tokenize` will contain a nonce and `error` will be `nil`; if it fails, `tokenize` will be `nil` and `error`will describe the failure.
+    @objc(tokenizeCard:completion:)
+    public func tokenize(_ card: BTCard, completion: @escaping (BTCardNonce?, Error?) -> Void) {
+        let request = BTCardRequest(card: card)
```

Comment:
Didn't we have analytic event sent here?

Reply:
That's in the `fpti-feature` branch which has not been merged into `master` yet. We will have to resolve the merge conflicts and update `fpti-feature` with the analytics events once this feature branch is merged into `master`.

Reply:
Since there are deletions of objective-c files where we had change in analytics, they wouldn't be conflicts but todo's?

Reply:
Not sure I follow, conflicts vs TODOs. In any case, we should be able to resolve it, will just involve looking at the diff and moving the events from the `.m` file to the `.swift` file. It will be _slightly_ more complicated but not impossible. We should though prioritize adding analytics for the modules already in Swift though to avoid similar issues in the future while we wrap up the conversions.